pinc/unicode.inc

```diff
@@ -96,14 +96,28 @@ function utf8_character_name($characters)
     return implode(" + ", $title_pieces);
 }
 
+function split_graphemes($string)
+{
+    $next = 0;
+    $maxbytes = strlen($string);
+
+    while ($next < $maxbytes)
+    {
+        $char = grapheme_extract($string, 1, GRAPHEME_EXTR_COUNT, $next, $next);
+        if (empty($char))
```

Comment:
This is the source of our missing-zero problem. From [PHP docs for empty()](https://www.php.net/manual/en/function.empty.php):
```
The following values are considered to be empty:
* "" (an empty string)
* 0 (0 as an integer)
* 0.0 (0 as a float)
* "0" (0 as a string)
* NULL
* FALSE
* array() (an empty array)
```

Are we certain this check is needed? I see it in all of the examples but I'm unclear on WHY it is needed it.

Reply:
I don't know either.

Reply:
I just updated to allow "0".

Reply:
The more I look at the `empty()` check in the PHP doc user notes the more I think it's just wrong and isn't needed at all.